### DISCUSSION TITLE
Recommended settings for generating a Medicare-specific population?

### DISCUSSION THREAD

**Discussion by telljmc** [synthetichealth/synthea#1536] [Q&A] [answered]
We're trying to generate a synthetic Medicare population that includes:

1. Primarily beneficiaries aged 65+
2. Appropriate proportion of under-65 beneficiaries with disabilities
3. Small percentage with ESRD
4. Mix of Medicare Advantage and Traditional Medicare

The current approach using basic age range settings doesn't generate a representative population group.

Questions:

- Are there specific configuration settings to generate a Medicare-focused population?
- What's the recommended approach for controlling Medicare eligibility factors (age, disability, ESRD)?
- Is there a way to set Medicare Advantage vs Traditional Medicare ratios?

**Comment by hadleynet** [comment, marked as answer]
Sounds like you are trying to do something similar to this:

https://data.cms.gov/sites/default/files/2023-05/d51e1218-68c3-4c7c-9598-0b81f22fe903/User%20Guide%20-%20CMS%20Synthetic%20RIF%20Files%20May%202023_AM508_v2.pdf

We do include a mix of Medicare Advantage and Traditional Medicare enrollment data but we don't model MA claims, only fee for service.

The script on this wiki page shows the Synthea settings used to generate this data set: https://github.com/synthetichealth/synthea/wiki/BFD-RIF-Export

**Reply by telljmc** [reply]
Yes, I would like the capability to segment the population into two groups, MA and FFS Medicare, but I would be fine with a population that has both.  My current perception may be incorrect, but I foresee at least two potential challenges to generating a Medicare population that resembles a real Medicare population.  

The first is it does not appear that I can simply set a configuration parameter and have Synthea return only Medicare patients.  For, if I set the age to 65+, I lose the patients that would normally qualify for Medicare because they are disabled or have ESRD and are younger than 65.

Secondly, it does not appear that one can easily generate a Medicare population in FHIR or CSV format as you have to link the BFD patient info to the other two file sets to capture the full spectrum of data elements.  Linking the file sets is not straightforward as it does not look like the UUID patient identifier in the FHIR and CSV datasets exists in the BFD files and I cannot find a crosswalk.  Therefore I anticipate that I will need to match based on patient name, DOB, gender, and possibly zip code to acquire the necessary Medicare data elements from the BFD files.  It also looks like I will have to build the required BFD mapping files in order to obtain a realistic data set.

Is that an accurate assessment or am I missing an easier path?

**Reply by hadleynet** [reply]
It's true that if you set the minimum age to 65 then you will lose patients that are younger and qualify for medicare via disability or ESRD. But note that the RIF exporter will only include claims for patients once they qualify for medicare and Synthea does support qualification via disability and ESRD.

It's also true that you'll have to do some record linking if your target is FHIR or CSV (though you can treat RIF as CSV with a '|' separator).

Finally, yes you will have to create some mapping files to obtain a realistic data set since the RIF export requires CPT/HCPCS and ICD instead of the native SNOMED codes used in Synthea. The RIF exporter will skip entries where no mapping is available. There is a configuration option (`exporter.bfd.export_missing_codes=true`) to output a file that lists all of the codes that couldn't be mapped.
